pkg/globalmanager/controllers/objectsearch/upstream.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2021 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package objectsearch
+
+import (
+	//"context"
+	"encoding/json"
+	//"fmt"
+
+	//sednav1 "github.com/kubeedge/sedna/pkg/apis/sedna/v1alpha1"
+	"github.com/kubeedge/sedna/pkg/globalmanager/runtime"
+	//metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+)
+
+/*
+func (c *Controller) updateMetrics(name, namespace string, metrics []sednav1.Metric) error {
+	client := c.client.ObjectSearchServices(namespace)
+
+	return runtime.RetryUpdateStatus(name, namespace, func() error {
+		joint, err := client.Get(context.TODO(), name, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+		joint.Status.Metrics = metrics
+		_, err = client.UpdateStatus(context.TODO(), joint, metav1.UpdateOptions{})
+		return err
+	})
+}
+*/
+
+// updateFromEdge syncs the edge updates to k8s
+func (c *Controller) updateFromEdge(name, namespace, operation string, content []byte) error {
```

Comment:
`updateFromEdge` can not be called since no update from LC.
Recommended that remove this untested implemention, and add a TODO comment.

Reply:
Removed and add TODO comment.